package.json

```diff
@@ -7,19 +7,21 @@
   },
   "scripts": {
     "start": "./node_modules/.bin/ts-node -r tsconfig-paths/register src/main/server.ts",
-    "cichecks": "yarn && yarn sass && yarn lint  && yarn test && yarn test:routes &&  yarn test:a11y",
+    "cichecks": "yarn && yarn build && yarn lint && yarn test && yarn test:routes && yarn test:a11y",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx && sass-lint -v -q",
-    "sass": "gulp sass copy-files",
+    "build": "yarn webpack",
+    "build:dev": "concurrently \"yarn run start\" \"yarn webpack-dev-server\"",
```

Comment:
@akomisarek we could just have the start script run webpack and server, thoughts?

Reply:
I think I would locally expect working server when I run yarn start. Currently I got:
Error: ENOENT: no such file or directory, stat '/Users/adamko/Work/FPLA/code/adoption-frontend/src/main/public/assets/images/favicon.ico'
so yes please, if that fixes it, go for it.

Reply:
Current implementation of build dev would not fix this. Issue is because webpack now moves the favicon from govuk-frontend assets from within node_modules. Start script will not be able to use concurrently.